source/_posts/2024-04-03-release-20244.markdown

```diff
@@ -121,7 +121,7 @@ types on the left that stay put and view the filtered results on the right.
 We also introduced multiple new filter types for each table. For example,
 {% term automations %} can now be filtered not only by {% term areas %},
 {% term entities %} , or {% term devices %}, but also by used
-{% term blueprints %}, labels, and categories (more on these later!).
+{% term blueprints "blueprints," %} labels, and categories (more on these later!).
```

Comment:
Is the double blueprints correct?

Reply:
Yes, it denotes the term to look up vs what to print on the page.

In this case, the `,` gets lost with an additional new-line on a desktop. Not sure why, never seen it before. This works around it.

Reply:
But it doesn't work so got to revert it :)